guidelines/index.html

```diff
@@ -443,15 +443,16 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
                					
                
                					
-               <p>In order to avoid confusion for implementers for whom backwards compatibility to WCAG 2 versions is important, new success criteria in WCAG 2.2 have been appended to the end of the set of success criteria within their guideline. This avoids the need to change the section number of success criteria from WCAG 2, which would be caused by inserting new success criteria between existing success criteria in the guideline, but it means success criteria in each guideline are no longer grouped by conformance level. 各ガイドライン内での達成基準の順番は、適合レベルに関する情報を暗示するものではなく、各達成基準自身の適合レベル表示 (A / AA / AAA) のみが適合レベルを示す。<a href="https://www.w3.org/WAI/WCAG22/quickref/">WCAG 2.2 クイックリファレンス</a>は、他の多数のフィルターや並べ替え方法と共に、適合レベルごとにグループ化された達成基準を見る方法を提供する。</p>
+               <p>WCAG 2 系のバージョンに対する後方互換性が重要となる開発者の混乱を招くことを避けるため、WCAG 2.2 における新しい達成基準はガイドライン内の一連の達成基準の末尾に追加されている。これにより、ガイドラインの既存の達成基準の間に新しい達成基準を挿入することによって生じるであろう、WCAG 2.1 からある達成基準のセクション番号の変更の必要性が、回避される。しかしながらこのことは、各ガイドライン内の達成基準がもはや適合レベルごとにグループ分けされていないことを意味する。各ガイドライン内での達成基準の順番は、適合レベルに関する情報を暗示するものではなく、各達成基準自身の適合レベル表示 (A / AA / AAA) のみが適合レベルを示す。<a href="https://www.w3.org/WAI/WCAG22/quickref/">WCAG 2.2 Quick Reference</a>は、他の多数のフィルターや並べ替え方法と共に、適合レベルごとにグループ化された達成基準を見る方法を提供するだろう。</p>
```

Comment:
```suggestion
               <p>WCAG 2 系のバージョンに対する後方互換性が重要となる開発者の混乱を招くことを避けるため、WCAG 2.2 における新しい達成基準はガイドライン内の一連の達成基準の末尾に追加されている。これにより、ガイドラインの既存の達成基準の間に新しい達成基準を挿入することによって生じるであろう、WCAG 2 からある達成基準のセクション番号を変更する必要がなくなる。しかしながらこのことは、各ガイドライン内の達成基準がもはや適合レベルごとにグループ化されていないことを意味する。各ガイドライン内での達成基準の順番は、適合レベルに関する情報を暗示するものではなく、各達成基準自身の適合レベル表示 (A / AA / AAA) のみが適合レベルを示す。<a href="https://www.w3.org/WAI/WCAG22/quickref/">WCAG 2.2 Quick Reference</a> は、他の多数のフィルター及び並べ替え方法とともに、適合レベルごとにグループ化された達成基準を見る方法を提供するだろう。</p>
```

「とともに」 [公⽤⽂における漢字使⽤等について](https://www.bunka.go.jp/kokugo_nihongo/sisaku/joho/joho/kijun/sanko/koyobun/pdf/kunrei.pdf)

少し変更しました。以下については手を入れていません。

- implementersは実装者のほうがよい？
- willを活かすなら、「は」を「が」に変えるのが適当かもしれない？